client/editorQueryEvaluator.ts

```diff
@@ -210,7 +211,8 @@ export class EditorQueryEvaluator {
         resultsEditorTabIdentifier: Uri,
         editor: TextEditor
     ): void {
-        const contentType: contentType = workspace.getConfiguration().get('marklogic.sparqlContentType');
+        const queryType: ml.GraphsQueryType = getSparqlQueryForm(sparqlQuery);
+        const contentType: ml.GraphsResponseFormat = getSparqlResponseType(queryType);
```

Comment:
Based on the docs - https://docs.marklogic.com/guide/semantics/REST#id_54258 - call this `SparqlOutputFormat`. The docs do use the term "Response Output Format" in the section above that, but that's a little redundant, and "output format" is well understood as it's used in other projects too for the same thing.